packages/block-library/src/post-navigation-link/index.php

```diff
@@ -14,7 +14,7 @@
  *
  * @return string Returns the next or previous post link that is adjacent to the current post.
  */
-function render_block_core_post_navigation_link( $attributes, $content, $block ) {
+function render_block_core_post_navigation_link( $attributes, $content, $block ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Wouldn't be better to remove the unused variables? 🤔  @swissspidy

Reply:
Depending on usage callbacks in PHP need to have the exact number of arguments they are called with to avoid warnings.

I think in this case it should be fine to remove. Otherwise you'll quickly see the warnings :)

I thought the block `render_callback` only has two arguments anyway 🤔 

https://github.com/WordPress/wordpress-develop/blob/1446948f95fcb4c8fa05cb3c7ac6c22ba5f6fb52/src/wp-includes/class-wp-block-type.php#L250

So where does `$block` come from? If it's `null` anyway sounds safe to drop.

Reply:
You're right, in this case the extra, unused args, could be removed 👍